racket/src/expander/demo.rkt

```diff
@@ -736,10 +736,24 @@
 
 (check-error
  (eval-module-declaration '(module m '#%kernel
-                            (#%require '#%kernel 
+                            (#%require '#%kernel
                                        'definition-shadows-initial-require)))
  #rx"already required")
 
+(eval-module-declaration '(module require-conflict-is-sourced-a '#%kernel
```

Comment:
These tests should maybe be in `pkgs/racket-test-core/tests/module.rktl` since this file is mostly a demo.

Reply:
Thanks! I've made the change.